pmd-java/src/main/java/net/sourceforge/pmd/lang/java/typeresolution/ClassTypeResolver.java

```diff
@@ -624,6 +627,83 @@ public Object visit(ASTType node, Object data) {
         return data;
     }
 
+    private void populateVariableDeclaratorFromType(ASTLocalVariableDeclaration node) {
+        ASTType type = node.getTypeNode();
+        // also assign this type to VariableDeclarator and VariableDeclaratorId
+        TypeNode var = node.getFirstChildOfType(ASTVariableDeclarator.class);
```

Comment:
This doesn't account for multiple variable declarations (part of #910 probably).

Reply:
Yeah you're right that went over my head